src/routes/user/show.rs

```diff
@@ -272,23 +338,33 @@ pub async fn get_item_sound_url(audio: AudioPlayback) -> Result<Vec<u8>, ServerF
 
 #[component]
 pub fn ShowUser() -> impl IntoView {
-    let params = use_params_map();
-    let user_id_string = params.read_untracked().get("id").unwrap_or_default();
-
-    let user_id = match user_id_string.parse::<i64>() {
-        Ok(id) => UserId(id),
-        Err(e) => {
-            return view! { <p class="text-red-500">"Failed to convert id to a number: "{e.to_string()}</p> }
-            .into_any();
+    let (user_id, set_user_id) = signal(None);
+    Effect::new(move |_| match get_user_id().get() {
+        Ok(ok) => set_user_id.set(Some(ok)),
+        Err(e) => set_error(e),
+    });
+    pub async fn load_user_or_none(
+        user_id: ReadSignal<Option<UserId>>,
+    ) -> Result<Option<User>, ServerFnError> {
+        match user_id.get() {
+            Some(user_id) => load_user(user_id).await,
+            None => {
+                //FIXME there must be a cleaner solution
+                let f = async move || Ok(None);
+                f().await
+            }
         }
-    };
+    }
 
-    let user_resource = OnceResource::new(get_user(user_id));
+    let user_resource = LocalResource::new(move || load_user_or_none(user_id));
```

Comment:
Why can this not just be a call to [`get_user`](https://github.com/strichliste-rs/strichliste-rs/pull/65/files#diff-8d274d2f473c80fd141dd8e773b8626b9ec0d1ab0244dd145595136b2b341bdeR40)